DevEdu.API/DevEdu.DAL/Repositories/TaskRepository.cs

```diff
@@ -18,8 +18,9 @@ public TaskRepository()
         }
         public TaskDto GetTaskById(int id)
         {
-            TaskDto task = _connection.QuerySingle<TaskDto>(
-                "dbo.Task_SelectById",
+            string query = "dbo.Task_SelectById";

```

Comment:
please move all the stored procedures' names to the top of the class (like it was in the lesson)